index.bs

```diff
@@ -241,6 +247,8 @@ An attribution source is a [=struct=] with the following items:
 :: An [=url/origin=].
 : <dfn>expiry</dfn>
 :: A point in time.
+: <dfn>priority</dfn>
+:: A 64-bit integer.
```

Comment:
Where is this going to be used? In a future PR for this spec?

Reply:
Yes, this is a pre-requisite for spec'ing the conversion/attribution algorithms